### DISCUSSION TITLE
Known issues

### DISCUSSION THREAD

**Discussion by qing762** [qing762/exitLag-auto-signup#4] [Announcements]
_<ins>Here are the known issues that I found:</ins>_
### Zero trial days.

- You are able to gain 3 days trial days by simply shrink your disk volume through disk management (Windows).

[FOR MORE SUPPORT JOIN MY DISCORD SERVER](https://qing762.is-a.dev/discord)

**Comment by SagMestKing** [comment]
How do you install Chrome and not Chromium?

**Reply by qing762** [reply]
https://google.com/chrome

**Reply by qing762** [reply]
Could be your ip is suspended. Can't really help with it.

For now, Google Chrome is the one and only way I found to bypass this. If that doesn't work for you, I can't really help out until a new solution is found ig?

**Reply by SagMestKing** [reply]
I mean it could be me having Chromium / ChromeDriver. How do I make sure I have ONLY Chrome installed?

**Reply by qing762** [reply]
The program will prioritize Chrome over Chromium. No need to worry about that.

**Comment by marc22225** [comment]
when I open the main.py it closes immediately

**Reply by qing762** [reply]
Instead of double clicking it, you should run the script file inside a command prompt session.
Here a guide for you: https://qing762.is-a.dev/exitlag-guide

**Comment by Dxnyy** [comment]
Im still getting the list index out of range

**Reply by qing762** [reply]
This is fixed along the new update with the dynamic wait implementation.

**Comment by jackammoka** [comment]
How to activate the account sir. all the account that had been made are already expire.

**Reply by qing762** [reply]
Use the latest version along with a HWID spoofer

**Comment by future1105** [comment]
it asks do you want to block ip address y/n, if I press y it says permission denied please run the program as adminitrator

**Reply by qing762** [reply]
As the text stated, you should run the script as an administrator
Also fyi youll get zero trial days upon logging in to exitlag application
A fix is on its way

**Comment by Forkinator** [comment]
I can't seem to get git to detect  chrome browser. 

[Error](https://imgur.com/nKg9yLA)

**Reply by qing762** [reply]
https://github.com/roniemartinez/browsers/issues/144
https://github.com/roniemartinez/browsers/pull/146
Known issue. Waiting for owner to accept pr actually.

**Comment by Pleasntly** [comment]
Getting this error, any ideas?

. This could take a while.
ERROR: Could not find a version that satisfies the requirement pywin32<307,>=303; sys_platform == "win32" (from pybrowsers) (from versions: 307, 308)

[notice] A new release of pip is available: 24.2 -> 24.3.1
[notice] To update, run: python.exe -m pip install --upgrade pip
ERROR: No matching distribution found for pywin32<307,>=303; sys_platform == "win32"

C:\Users\tyhes\exitLag-auto-signup>

**Reply by qing762** [reply]
Hey there sorry for the late reply. But feel free to install the executable. It should run the script without relying on Python and libraries as it is already compiled into the executable.
